src/Sulu/Bundle/AdminBundle/Resources/js/components/PublishIndicator/publishIndicator.scss

```diff
@@ -4,6 +4,15 @@ $indicatorDraftBackgroundColor: $gold;
 $indicatorPublishedBackgroundColor: $mantis;
 $indicatorBorderColor: $white;
 $indicatorDimension: 18px;
+$indicatorOverlap: 10px;
+
+.publish-indicator {
+    min-width: $indicatorDimension;
+
+    &.has-published.has-draft {
+        min-width: calc($indicatorDimension * 2 - $indicatorOverlap);
```

Comment:
@danrot Instead of calculation I would go with the following solution:

```diff
.publish-indicator {
+     white-space: nowrap;
}

.published,
.draft {
+     display: inline-block;
-     display: block;
-     float: left;
}
```

or setting the wrap container to flex

```diff
.publish-indicator {
+     display: flex;
}
```

Reply:
I tried `white-space: nowrap`, but it didn't work... Didn't know that I've had to change the `display` property of the children as well... About to push :slightly_smiling_face: